esx_service/utils/auth_data.py

```diff
@@ -561,10 +567,46 @@ def handle_upgrade_1_1_to_1_2(self):
                      """
             sql_script = script.format(DB_MAJOR_VER, DB_MINOR_VER)
             self.conn.executescript(sql_script)
+
+            # check if _DEFAULT tenant exist, if not, insert _DEFAULT tenant
```

Comment:
If user has explicitly deleted default tenant ( for tighter access control), is it correct to create it again?